config/ldes-delta-pusher/handle-decision-type.ts

```diff
@@ -1,27 +1,34 @@
-import { query } from "mu";
```

Comment:
use sudo queries as we need to trigger dispatch from the ldes-delta-pusher's cron for healing so we don't have a session to use for regular query.

Reply:
this happens in other files too